AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -217,6 +220,21 @@ - (void)fetchData
       _player = [[AVPlayer alloc] initWithPlayerItem:_currentItem];
       _player.muted = _muted;
     }
+    __weak __typeof(self) weakSelf = self;
+    _timeObserver = [_player addPeriodicTimeObserverForInterval:CMTimeMake(1, 1) queue:NULL usingBlock:^(CMTime time){
```

Comment:
This is awesome!  Is it possible to decrease the interval of the periodic time observer to like 1/100 sec?   This will help smoothen the scrubber UI update.